docker/push

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -e
+HERE=$(dirname $0)
+. $HERE/common
+
+# In case we switch agents between steps
+[ ! -z $(docker images -q $TAG_SHA) ] || docker pull $TAG_SHA
+
+docker tag $TAG_SHA $TAG_BRANCH
+docker push $TAG_BRANCH
+
+if [ $GIT_BRANCH == "master" ]; then
+    docker tag $TAG_SHA $TAG_LATEST
+    docker tag $TAG_SHA $TAG_VERSION
+    docker push $TAG_VERSION
+    docker push $TAG_LATEST
```

Comment:
what purpose does this latest tag serve, isn't it just equivalent to master?

Reply:
it means that `docker pull mrcide/mintr` will work, as `latest` is the standard docker convention as `master` is on git